administrator/modules/mod_guidedtours/tmpl/default.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  mod_guidedtours
+ *
+ * @copyright   (C) 2019 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+defined('_JEXEC') or die;
+
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+
+// Load the Bootstrap Dropdown
+HTMLHelper::_('bootstrap.dropdown', '.dropdown-toggle');
```

Comment:
can we use the web asset manager please

Reply:
That is what HTMLHelper::_('bootstrap.dropdown', '.dropdown-toggle'); does. It calls the asset manager